ddp-workspace/projects/ddp-sdk/src/lib/components/cookies/cookiesBanner/cookiesBanner.component.ts

```diff
@@ -0,0 +1,69 @@
+import { Component, Inject } from '@angular/core';
+import { CookiesManagementService } from '../../../services/cookiesManagement.service';
+import { MatDialog } from '@angular/material/dialog';
+import { ConfigurationService } from '../../../services/configuration.service';
+import { NoopScrollStrategy } from '@angular/cdk/overlay';
+import { CookiesPreferencesModalComponent } from '../cookiesPreferencesModal/cookiesPreferencesModal.component';
+import { PrivacyPolicyModalComponent } from '../privacyPolicy/privacyPolicyModal.component';
+
+@Component({
+  selector: 'ddp-cookies-banner',
+  template: `
+  <div class="cookiesBanner">
+    <div class="cookiesBanner__text">
+      <span translate>SDK.CookiesBanner.Message_First</span>
+      <span translate>SDK.CookiesBanner.Message_Second</span>
+      <button class="cookiesBanner__link policy"
+              (click)="openPolicy()"
+              [innerText]="'SDK.CookiesBanner.Policy' | translate"></button>
+      <span translate>SDK.CookiesBanner.Message_Third</span>
+    </div>
+    <button class="cookiesBanner__link"
+            (click)="openPreferences()"
+            [innerText]="'SDK.CookiesBanner.Preferences' | translate"></button>
+    <button class="Button Button--borderedYellow col-lg-4 col-md-4 col-sm-8 col-xs-8"
```

Comment:
Rather than using Button--borderedYellow specifically, it might be good to use something more generic so that other studies can adapt it to their color scheme.  Something like CookieButton--Reject, maybe.

Reply:
Done in 527818e